src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
```

Comment:
The class 'ColorSupplier' should not start with an empty line. Please remove the empty line at the beginning of the class definition.